perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -865,6 +878,24 @@ def Cleanup(self, vm):
     benchmark_spec_class.AZURE: AzureBlobStorageBenchmark()}
 
 
+WRONG_REGION_FLAGS = {
```

Comment:
I'm trying to guard against the situation where you run the benchmark with one storage provider, then switch to another storage provider, but forget to say what region you want to use with the new provider. This is a "hey, you probably haven't updated your command line all the way" message.